search-app/src/cmr/search/results_handlers/subscriptions_json_results_handler.clj

```diff
@@ -21,13 +21,15 @@
           deleted :deleted
           provider-id :provider-id
           native-id :native-id
+          subscription-type :subscription-type
           concept-id :concept-id} :_source} elastic-result
         revision-id (elastic-results/get-revision-id-from-elastic-result :service elastic-result)
         result-item (util/remove-nil-keys
                      {:concept_id concept-id
                       :revision_id revision-id
                       :provider_id provider-id
                       :native_id native-id
+                      :subscription-type subscription-type
```

Comment:
We want to change `:subscription-type` to `:type`. As the ticket asking for `type` field to be added.

Reply:
Fixed